musicbrainz/csv/get_genre.py

```diff
@@ -0,0 +1,64 @@
+"""
+Script for retrieving the "genre" dumps.
+"""
+
+import time
+import requests
+from requests.exceptions import HTTPError
+import pandas as pd
+from bs4 import BeautifulSoup
+
+URL = "https://musicbrainz.org/ws/2/genre/all"
+PARAMS = dict(fmt="json", limit="50")
+MAX_REQUEST_RETRIES = 3
+
+data = []
+max_records = requests.get(url=URL, params=PARAMS, timeout=50).json()["genre-count"]
+
+for i in range(0, max_records, 50):
+    PARAMS["offset"] = i
+    for j in range(MAX_REQUEST_RETRIES):
+        try:
+            resp = requests.get(url=URL, params=PARAMS, timeout=50)
+            resp.raise_for_status()
+            break
+
+        except HTTPError as exc:
+            code = exc.response.status_code
+
+            if code == 503:
+                if j == 2:
+                    raise
+                time.sleep(j + 1)
+                continue
+
+            raise
```

Comment:
What's this doing here? I think you could get rid of `raise` here and `break` on line 24.

Reply:
This is because HTTPError is not only when the server returns code 503, when it is unavailable. If other codes are returned and it is a HTTPError, then we do not handle it and raise that exception directly.